cloudscale/resource_cloudscale_server.go

```diff
@@ -113,6 +113,22 @@ func getServerSchema() map[string]*schema.Schema {
 			},
 			Computed: true,
 		},
+		"public_ipv4": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"public_ipv6": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"private_ipv4": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"private_ipv6": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
```

Comment:
I feel like we should call it `public_ipv4_address` and so on. I feel like others like digitalocean and others are doing that as well (and IMO it's the correct terminology).

Also: We definitely don't need `private_ipv6_address`, it's not necessary, because it will never be set. The private network v4 address could be useful, but I'm not sure if it's clever to include it.

The question is then arises if we should use `ipv4_address` instead, which just returns the first address (public if possible, otherwise private). Since this is a helper attribute anyway, I feel like we don't need to return everything.

What do you think? Do you have a use case for the private IPv4 address and the public one?

Reply:
Hi Dave,

thanks for the feedback. I can reword it to `public_ipv4_address` and remove the private v6 address.

We're deploying DNS records for both the public and private addresses, that's why I would like to keep the reference to the private one. 

At the moment, we're using the DNS record for the private address when we deploy kubernetes clusters and configure DNS of the private address in the "hostname-override" field in kubelet. 

Please let me know if you think it is okay to keep the private address.

Reply:
Hi Ferdinand. I'm now 100% fine with the changes, except that you forgot two things. Please fix those and we're good to merge.